test/CMakeLists.txt

```diff
@@ -57,10 +57,10 @@ set(MIOPEN_TEST_FLOAT_ARG)
 
 if(MIOPEN_TEST_HALF)   
     if(MIOPEN_BACKEND_OPENCL)
-        set(SKIP_TESTS test_gru test_rnn_vanilla test_lstm)
+        set(SKIP_TESTS test_gru test_rnn_vanilla test_lstm test_conv_for_dynamic_implicit_gemm)
```

Comment:
Why? iGemm solvers are normally working fine with OCL backend.

Reply:
Hi, this is the ctest dedicated for kernel in src/kernels/composable_kernel/asm/igemm_v4r1_dynamic.s of this PR, and currently it is only used for fp32. So, I just ignore it when testing HALF

Reply:
I see, thanks.